### PR TITLE
Addition of a right

### PR DESCRIPTION
**Pull request by joinimran**
A right was missing from the list. Added the basic info. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8551

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by JohanFreelancer9 — COMMENTED** [review]
Hi Imran. Seems that something went wrong and the link is not inserted. Could you please insert the link? Thanks.

File: windows/security/threat-protection/security-policy-settings/user-rights-assignment.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ The following table links to each security policy setting and provides the const
 | [Manage auditing and security log](manage-auditing-and-security-log.md)| SeSecurityPrivilege| 
 | [Modify an object label](modify-an-object-label.md) | SeRelabelPrivilege| 
 | [Modify firmware environment values](modify-firmware-environment-values.md)| SeSystemEnvironmentPrivilege| 
+| [Obtain an impersonation token for another user in the same session]) | SeDelegateSessionUserImpersonatePrivilege|
```
Comment (illfated) [inline windows/security/threat-protection/security-policy-settings/user-rights-assignment.md:72]:
```suggestion
| [Obtain an impersonation token for another user in the same session](impersonate-a-client-after-authentication.md) | SeDelegateSessionUserImpersonatePrivilege|
```

Comment (illfated) [reply windows/security/threat-protection/security-policy-settings/user-rights-assignment.md:72]: I am reasonably sure this was the intended file link, but you want to verify that it is the correct one:
- [`public` windows/security/threat-protection/security-policy-settings/impersonate-a-client-after-authentication.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/security-policy-settings/impersonate-a-client-after-authentication.md)

**Comment by joinimran** [comment]
@illfated - Thanks, how did you find this file.

**Comment by illfated** [comment]
Well, first of all I opened the same directory as the file used in your PR, `user-rights-assignment.md`
- https://github.com/MicrosoftDocs/windows-itpro-docs/tree/public/windows/security/threat-protection/security-policy-settings

Secondly, I searched for some of the keywords in the security setting (` SeDelegateSessionUserImpersonatePrivilege `) 
and your added description ("Obtain an impersonation token for another user in the same session")

By removing the word ending, the shortened word form `impersonat` led me to the file:
- `SeDelegateSessionUserImpersonatePrivilege`
- "Obtain an impersonation token [ ... ]"

Noticing that this keyword might help me find a useful file, and the fact that all the other columns in the table contains links to files in this same directory (https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/security-policy-settings/impersonate-a-client-after-authentication.md), I searched the filename list for the keyword `impersonat` and found:
- [`impersonate-a-client-after-authentication.md`](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/security-policy-settings/impersonate-a-client-after-authentication.md)

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by Damien-Sugden** [comment]
Hi, The right is listed, but it links to the wrong content. It links to Impersonate a client after authentication which is a different right altogether.

**Pull request merged**